File: Source/Fantasy.Jobs/Tasks/CallTargetTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs.Tasks
{
    [Task("callTarget", Consts.XNamespaceURI, Description = "Invoke specified target immediately")] 
    public class CallTargetTask : ObjectWithSite, ITask
    {
        #region ITask Members

        public bool Execute()
        {
            IJob job = this.Site.GetRequiredService<IJob>();
            if(this.Targets != null)
            {
                foreach(string target in this.Targets)
                {
                    if(!string.IsNullOrEmpty(target))
                    {
                        job.ExecuteTarget(target);
                    }
                }
            }

            return true;
             
        }
        #endregion

        [TaskMember("target", Description = "The target or targets to execute")] 
        public string[] Targets { get; set; }
    }
}

File: AutoConverted/Fantasy.Jobs/Tasks/CallTargetTask.java
package Fantasy.Jobs.Tasks;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[Task("callTarget", Consts.XNamespaceURI, Description = "Invoke specified target immediately")]
public class CallTargetTask extends ObjectWithSite implements ITask
{
//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#region ITask Members

	public final boolean Execute()
	{
		IJob job = this.Site.<IJob>GetRequiredService();
		if(this.getTargets() != null)
		{
			for(String target : this.getTargets())
			{
				if(!DotNetToJavaStringHelper.isNullOrEmpty(target))
				{
					job.ExecuteTarget(target);
				}
			}
		}

		return true;

	}
//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#endregion

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[TaskMember("target", Description = "The target or targets to execute")]
	private String[] privateTargets;
	public final String[] getTargets()
	{
		return privateTargets;
	}
	public final void setTargets(String[] value)
	{
		privateTargets = value;
	}
}